### PR TITLE
Added Clustering functionality

### PR DESCRIPTION
**Pull request by mazlixek**
I added clustering functionality, I tried it in my app and it is working.

### CODE REVIEW

File: gwt-ol3-client/src/main/java/ol/source/Cluster.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package ol.source;
+
+import jsinterop.annotations.JsProperty;
+import jsinterop.annotations.JsType;
+
+import ol.source.Vector;
+import ol.Feature;
```
Comment (TDesjardins) [inline gwt-ol3-client/src/main/java/ol/source/Cluster.java]: Unused import?

**Comment by TDesjardins** [comment]
O.k. Thanks!

**Comment by mazlixek** [comment]
Just so you know, I didn't test the example as I have no idea how to run it, but It is inspired by this [example](http://openlayers.org/en/v3.6.0/examples/cluster.html) so it should work. I don't see it in the live example now... So I'm not sure if I didn't forget to add it somewhere, I only created the Java file with the example.

**Comment by TDesjardins** [comment]
Your example works. You can run it with maven. Just use `mvn gwt:devmode`. The live example have to be updated.

**Pull request merged**